test/controllers/user.test.js

```diff
@@ -74,7 +93,7 @@ describe('usersController.screateUserignIn', () => {
   beforeEach(() => {
     userAttributes.email = 'omar.rodriguez@wolox.com';
     userAttributes.password = 'password1923';
-    return createUser();
+    return signUpUser();
```

Comment:
See https://github.com/wolox-training/adl-express-js/pull/8#discussion_r353746872 for a correction on this.